fcrepo-clients/src/test/java/edu/unc/lib/dl/event/RepositoryPremisLoggerIT.java

```diff
@@ -53,15 +53,16 @@
 
     @Before
     public void init() throws Exception {
```

Comment:
The tests from this class are failing for me, gives a 403 when creating deposit record